src/Sarif/Baseline/ResultMatching/SarifLogMatcher.cs

```diff
@@ -157,7 +157,7 @@ private List<ExtractedResult> GetMatchingResultsFromRuns(IEnumerable<Run> sarifR
                 {
                     foreach (Result result in run.Results)
                     {
-                        Rule rule = GetRuleFromResources(result, run.Resources.Rules);
+                        IRule rule = GetRuleFromResources(result, run.Resources.Rules);
```

Comment:
>IRule [](start = 24, length = 5)

I thought we agreed to revert all `Rule` => `IRule` changes, and then implement `IRule` => `Rule` in a separate PR.

Reply:
this rule instance isn't even used, which is how it remained out of sight. @evmaus, why is this code getting called? unnecessary code?

---
In reply to: [221404351](https://github.com/Microsoft/sarif-sdk/pull/1103#discussion_r221404351) [](ancestors = 221404351)